src/model/BlindedSignature.ts

```diff
@@ -1,19 +1,35 @@
 import { ProjPointType } from '@noble/curves/abstract/weierstrass';
 import { SerializedBlindedSignature } from './types/index.js';
+import { DLEQ } from '@cashu/crypto/modules/common';
+import { bytesToHex } from '@noble/hashes/utils.js';
 
 class BlindedSignature {
 	id: string;
 	amount: number;
 	C_: ProjPointType<bigint>;
+	dleq?: DLEQ;
 
-	constructor(id: string, amount: number, C_: ProjPointType<bigint>) {
+	constructor(id: string, amount: number, C_: ProjPointType<bigint>, dleq?: DLEQ) {
 		this.id = id;
 		this.amount = amount;
 		this.C_ = C_;
+		this.dleq = dleq;
 	}
 
 	getSerializedBlindedSignature(): SerializedBlindedSignature {
-		return { id: this.id, amount: this.amount, C_: this.C_.toHex(true) };
+		return {
+			id: this.id,
+			amount: this.amount,
+			C_: this.C_.toHex(true),
+			dleq:
```

Comment:
Can we use the spread syntax here too?

Reply:
Yes. Also I am responding here to your other comment about "why would someone strip the DLEQs when sending to Carol". One might try to get a smaller size cashuB token for example. In general if it's not required by Carol why include it?